lms/djangoapps/instructor/tests/test_api.py

```diff
@@ -869,7 +869,7 @@ def test_user_with_retired_email_in_csv(self):
         user.is_active = False
         user.save()
 
-        csv_content = b"{email},{username},tester,USA".format(email=conflicting_email, username='new_test_student')
+        csv_content = "{email},{username},tester,USA".format(email=conflicting_email, username='new_test_student')
```

Comment:
Prefix with a `u` if it's explicityl supposed to be unicode in both python 2 and 3.

Reply:
`csv_content` isn't supposed to be Unicode, it fails if converted to Unicode.

Reply:
If it's not supposed to be unicode, then why remove the `b` prefix?  The way you have it now, it will be bytes in python2 but unicode in python3 is that what we want?

Reply:
we removed `b` prefix to make it work with `format` method, 
on next line `SimpleUploadedFile("temp.csv", six.b(csv_content))`, `six` takes care of string input across python versions and if we prefix the input with `u` method `six.b`fails with 
'unicode' does not have the buffer interface
>

Reply:
Ah, ok, thanks for the clarification.